crates/planner/src/logical/errors.rs

```diff
@@ -89,6 +89,14 @@ pub enum Unsupported {
     UnqualifiedProjectExpr,
 }
 
+#[derive(Error, Debug)]
+#[error("Inserting a row with {values} values into {table} which has {fields} fields")]
+pub struct InsertError {
+    pub table: String,
+    pub values: usize,
+    pub fields: usize,
+}
+
```

Comment:
Will be nice to know which `fields`/`values` failed here.

Reply:
I added a TODO to that effect